infrastructure/index.ts

```diff
@@ -59,6 +59,14 @@ const originBucket = pulumi.output(aws.s3.getBucket({
     bucket: originBucketName,
 }));
 
+// Create a bucket to store large files.
+const largeFileBucket = new aws.s3.Bucket("large-file-bucket", {
```

Comment:
Technically fine, but I'd suggest a more generic name, since we needn't use this solely for "large files", but really any file we want to be able to serve and manage by hand, outside of Git. Maybe something like "ad-hoc" or "files" or "uploads" or something. `uploads` might be good, yeah, since it's generic and connotes a separate activity and pretty much is what we're doing as well. What do you think?

Reply:
`uploads` works great for me. One thing to keep in mind is the assets in the Bucket need to live under a `uploads` folder in the bucket. You might already know that, but that tripped me up for like 10 minutes when setting this up.

Reply:
Yep, makes sense and I think that's fine.